### PR TITLE
Move onboarding to the stack

### PR DESCRIPTION
**Pull request by maximemenager**

### CODE REVIEW

**Comment by maximemenager** [comment]
/!\ Do not merge before `coclyco` is updated to reflect these changes.

File: web/auth/auth.go [hunk 1]
```diff
@@ -1166,6 +1183,7 @@ func Routes(router *echo.Group) {
 	router.POST("/passphrase_reset", passphraseReset, noCSRF)
 	router.GET("/passphrase_renew", passphraseRenewForm, noCSRF)
 	router.POST("/passphrase_renew", passphraseRenew, noCSRF)
+	router.GET("/passphrase", passphraseForm, noCSRF)
```
Comment (nono) [inline web/auth/auth.go]: You know the rules: if you add a route, you need to add documentation for it. And for things in `/auth`, you need to have a very good reason to not add a unit test.

File: assets/templates/passphrase_onboarding.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+        <form id="myForm" method="POST" action="/settings/passphrase" onsubmit="submitform(event)">
+            <p>Choisissez votre password</p>
+            <input type="hidden" name="register_token" value={{.RegisterToken}}>
+            <input type="password" name="passphrase"><br><br>
+            <input type="submit" value="Submit">
+        </form>
```
Comment (nono) [inline assets/templates/passphrase_onboarding.html]: [nitpick] the indentation looks broken

File: assets/templates/passphrase_onboarding.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+        <form id="myForm" method="POST" action="/settings/passphrase" onsubmit="submitform(event)">
```
Comment (nono) [inline assets/templates/passphrase_onboarding.html]: Why do you use JavaScript here?

File: assets/templates/passphrase_onboarding.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            <p>Choisissez votre password</p>
+            <input type="hidden" name="register_token" value={{.RegisterToken}}>
+            <input type="password" name="passphrase"><br><br>
+            <input type="submit" value="Submit">
+        </form>
+
+</body>
+<script type="text/javascript">
+    function submitform(e){
```
Comment (nono) [inline assets/templates/passphrase_onboarding.html]: I don't think that an inline script will work with our Content Security Policy.

File: assets/templates/passphrase_onboarding.html [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+        e.preventDefault();
+        var ab = document.getElementById("myForm");
+        var formData = new FormData(ab)
+
+        var object = {};
+        formData.forEach(function(value, key){
+            object[key] = value;
+        });
+        var json = JSON.stringify(object);
+        console.log(json);
```
Comment (nono) [inline assets/templates/passphrase_onboarding.html]: [nitpick] Remove the `console.log`

File: assets/templates/passphrase_onboarding.html [hunk 6]
```diff
@@ -0,0 +1,42 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
```
Comment (nono) [inline assets/templates/passphrase_onboarding.html]: `Document` as a title?

File: assets/templates/passphrase_onboarding.html [hunk 7]
```diff
@@ -0,0 +1,42 @@
+<html lang="en">
```
Comment (nono) [inline assets/templates/passphrase_onboarding.html]: No `doctype`? No CSS?

**Comment by nono** [comment]
I have rebased master to fix the conflicts on the built assets.

**Comment by nono** [comment]
[Revoir la page de login sur android](https://trello.com/c/zKC3FgMw/922-revoir-la-page-de-login-sur-android)

**Pull request merged**